en/resources/tutorials/templating.md

```diff
@@ -41,9 +41,14 @@ import PackageDescription
 let package = Package(
     name: "myTemplates",
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.0.0")),
-        .package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", .upToNextMinor(from: "1.0.0")),
-        .package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", .upToNextMinor(from: "1.0.0"))
+        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.2.0")),
```

Comment:
For it to be possible to cut/paste this as a Package.swift file, the very first line needs to be:
```
// swift-tools-version:4.0
```

Reply:
I've added this as you suggested.